internal/infrastructure/kubernetes/ratelimit_deployment_test.go

```diff
@@ -88,6 +90,14 @@ func TestCreateOrUpdateRateLimitDeployment(t *testing.T) {
 				},
 			}
 			require.NoError(t, kube.Client.Get(context.Background(), client.ObjectKeyFromObject(actual), actual))
+
+			sortEnv := func(env []corev1.EnvVar) {
+				sort.Slice(env, func(i, j int) bool {
+					return env[i].Name > env[j].Name
+				})
+			}
+			sortEnv(tc.want.Spec.Template.Spec.Containers[0].Env)
```

Comment:
env is a list, why do we need to sort

Reply:
ref https://github.com/envoyproxy/gateway/pull/1363#discussion_r1177129324